core/src/epicli/data/common/defaults/configuration/postgresql.yml

```diff
@@ -2,6 +2,8 @@ kind: configuration/postgresql
 title: PostgreSQL
 name: default
 specification:
+  configuration:
+    password_encryption: md5  # md5 or scram-sha-256
```

Comment:
Why by default we don't have this set to scram?

Reply:
I wanted to left it as it is since for upgrade existing clusters (with existing md5 passwords) that might be difficult to force users to re-create passwords. I think this should be applied very intentionally by users. But maybe when all upgrade procedure will be ready we can finally decide what is better.